dll/cpl/sysdm/general.c

```diff
@@ -203,11 +203,12 @@ LRESULT CALLBACK RosImageProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam
                     AlphaBlend(hCreditsDC, 0, 0, pImgInfo->cxSource, pImgInfo->cySource, hLogoDC, 0,  0, pImgInfo->cxSource, pImgInfo->cySource, BlendFunc);
                     AlphaBlend(hCreditsDC, 0, offset, pImgInfo->cxSource, pImgInfo->cySource, hLogoDC, 0,  0, pImgInfo->cxSource, pImgInfo->cySource, BlendFunc);
 
-                    DeleteDC(hLogoDC);
-                    DeleteDC(hCreditsDC);
-                    ReleaseDC(NULL, hDC);
-
                     timerid = SetTimer(hwnd, 1, ANIM_TIME, NULL);
+
+                Cleanup:
```

Comment:
No indentation on labels.

Reply:
OK, i was not aware of this rule. I fix.